gcc/testsuite/gcc.target/riscv/rvp32/builtin-rvp32-add64.c

```diff
@@ -0,0 +1,36 @@
+/* This is a test program for add64 instruction.  */
+/* { dg-do compile { target riscv32*-*-* } } */
+/* { dg-options "-march=rv32gc_zpn_zpsfoperand -mabi=ilp32d -O3" } */
+/* { dg-final { check-function-bodies "**" "" "" } } */
+
+#include <rvp_intrinsic.h>
+
+
+/*
+**f0:
+** add64\ta[0-9],a[0-9],a[0-9]
+** ...
+*/
+
+int64_t f0 (int64_t x0, int64_t x1){
+
+    return __rv_sadd64(x0, x1);
```

Comment:
riscv c api format changed to `__riscv_`
https://github.com/riscv-non-isa/riscv-c-api-doc/pull/31

Reply:
I'm not sure should to follow the spec of p-ext format (https://github.com/riscv/riscv-p-spec/blob/master/P-ext-proposal.adoc#73-add64-64-bit-addition) or the one on this page(https://github.com/riscv-non-isa/riscv-c-api-doc/pull/31) ?

Reply:
Leave it as it is for now, and wait for the new p-ext proposal to come out before deciding

Reply:
I'm not sure about it now. Maybe as a existed format, should follow the spec.